### PR TITLE
Feature/add code flash

### PR DESCRIPTION
**Pull request by mogimogitomato**
- Add code flash function, and execute it when run codes.
- Add configuration class about flash colors, and update README.
- Add extension's debug settings(after `npm install` , open repository and press `F5` extension run as debug mode.)

Sample is  here.
Sample

#16

### CODE REVIEW

**Comment by mactkg** [comment]
Thanks, I'll check this PR tomorrow!

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "npm",
+			"script": "watch",
+			"problemMatcher": "$tsc-watch",
+			"isBackground": true,
```
Comment (mactkg) [inline .vscode/tasks.json:10]: You added this `tasks.json` because we want to run `npm watch` in background, right?

**Review by mactkg — COMMENTED** [review]
What a cool feature, thanks!
Codes looks good to me, but I refactored some places of code and submited pull request to your repository: https://github.com/mogimogitomato/vscode-sonic-pi/pull/1
Can you check it out the PR?

**Comment by mogimogitomato** [comment]
> Can you check it out the PR?

Thank you! I'll check soon.


>  You added this tasks.json because we want to run npm watch in background, right?

Yes!
By adding tasks.json,  `npm run watch`
 (this `watch` is defined in package.json's script `tsc -watch -p`) execute in background.
And when compile error detected, tasks report problems in console. 
https://code.visualstudio.com/docs/editor/tasks#_processing-task-output-with-problem-matchers

＊I referenced those debug construction from vscode's extension sample.
https://github.com/microsoft/vscode-extension-samples/tree/master/completions-sample

**Comment by mogimogitomato** [comment]
Thank you! I merged refactor PR, fixed lightweight point and resolve package-locked.json's conflict.
Best regards.

**Comment by mactkg** [comment]
Thanks!

**Pull request merged**